docker-compose.vs.debug.yml

```diff
@@ -1,20 +0,0 @@
-version: '2'
```

Comment:
How come we dont need this any more?

Reply:
It's been replaced by version: '3'

The whole update process is something visual studio does automatically.